src/ui/popup/site-list-page/index.tsx

```diff
@@ -28,6 +29,7 @@ export default function SiteListPage(props: ViewProps) {
                 onChange={onSiteListChange}
             />
             <label class="site-list-page__description">Enter website name and press Enter</label>
+            <RemoveAllButton {...props} />
```

Comment:
I'm not sure if we should add another button here, it's already quite crowded in my opinion. What about making this a trash icon, which would be added on the top-{left,right} of the site-list?

Reply:
That sounds great but about the exact position, do you think it's okay to add it next to the 'Disable on these websites' label or somewhere else? Maybe bellow it?

Reply:
> do you think it's okay to add it next to the 'Disable on these websites' label or somewhere else? Maybe bellow it?

I really would prefer if this icon will be on the site-list, my idea was roughly this:
![image](https://user-images.githubusercontent.com/25481501/191237247-211a5a22-08be-4a0e-bee1-ed7722230ba6.png)
Whereby the icon would be either on one of the two red spots.

Reply:
I updated the button. Let me know if there's anything I should change c: